src/warplib/implementations/maths/add.ts

```diff
@@ -11,26 +11,22 @@ import {
 
 export function add(): WarplibFunctionInfo {
   const fileName = 'add';
-  const imports = [
-    'from starkware.cairo.common.math_cmp import is_le_felt',
-    'from starkware.cairo.common.uint256 import Uint256, uint256_add',
-  ];
+  const imports: string[] = [];
```

Comment:
We can delete this variable if it isn't doing anything

Reply:
its necessary to be able to return that way, otherwise would be necessary create a variable for the returned element and both ways are kind of the same

Reply:
I think that it is a declaration with no use, and provides no value until the last line of code. I think it's better to just return empty list as part of the import:
```ts
return {fileName, imports: [], functions};
```